#22763 Remove unnecessary lines

erickgonzalez opened the pull request:
These lines aren't necessary, when copying the page the content is being related successfully. This creates duplicated entries with the wrong relation_type.

erickgonzalez committed:
#22763 Remove unnecessary lines

nollymar reviewed (changes requested):
A test should be implemented

alfredo-dotcms committed:
dotCMS/core#22566 fix Improvements for ENTER key event (#22821)

* dotCMS/core#22566 fix Improvements for ENTER key event

* PR Feedback

victoralfaro-dotcms committed:
CI: bumps version to 22.9.0-rc.11 [skip ci]

rjvelazco committed:
dotCMS/core#22165 Increase the base font size (#22890)

* fix: block-editor content base size

* fix: spacing

victoralfaro-dotcms committed:
CI: bumps version to 22.9.0-rc.12 [skip ci]

alfredo-dotcms committed:
dotCMS/core#22810 fix Contentlet editor behaving oddly (styles broken… (#22828)

* dotCMS/core#22810 fix Contentlet editor behaving oddly (styles broken?) in 22.09.
A revert of styles in dijit.css file was done, there is no clear explanation from the author why this was changed, more information on https://github.com/dotCMS/core/issues/22810

* added doc explaining why css class was reverted

jcastro-dotcms committed:
https://github.com/dotCMS/core/issues/22196

* #22196 : Replacing deprecated LibSass with Dart SASS Compiler v1.54.4

* #22196 : Fixing code that broke Unit Test.

* #22196 : Implementing changes requested via Code Review.

* #22196 : Code Review - Replacing wildcard imports with actual imports.

* #22196 : Fixing script for Linux ARM64 environments.

* #22196 : Fixing incorrect arch name.

* #22196 : Testing Dart SASS fix for Graviton instances.

* #22196 : Removing unnecessary script.

rjvelazco committed:
dotCMS/core#21866 BlockEditor - Add Link within dotCMS (#22879)

* fix: internal QA v1

* fix: internal QA v2

* fix: internal QA v3

* remove: autolink

* update dotcms-block-editor.js

victoralfaro-dotcms committed:
CI: bumps version to 22.9.0-rc.13 [skip ci]

victoralfaro-dotcms committed:
Using dotcms-seed:22.09 (#22900)

victoralfaro-dotcms committed:
Adding server config to dotCMS docker image as well (#22923)

jdotcms committed:
dotCMS/core#21244 Refactor feedback (#22878)

Commit:
Merge branch 'release-22.09' of github.com:dotCMS/core into issue-22763

Commit:
Merge branch 'master' of github.com:dotCMS/core into issue-22763

Commit:
#22763 Implementing IT
